src/kv/MemDB.cc

```diff
@@ -477,17 +499,17 @@ int MemDB::MDBWholeSpaceIteratorImpl::seek_to_first(const std::string &k)
   } else {
     m_iter = m_btree_p->lower_bound(k);
   }
-
+  
   if (m_iter == m_btree_p->end()) {
     return -1;
   }
+  fill_current();
```

Comment:
these two changes are fixing an unrelated bug in seek_to_*, right?  Can you put them in a separate commit?